### DISCUSSION TITLE
Locate Windows Notebook

### DISCUSSION THREAD

**Discussion by CrazYoshi** [ESPresense/ESPresense#1268] [Q&A] [answered]
Hi,
I would like to know if it is possible to locate my Windows 11 Notebook using ESPresence. 
I've tried to check the Fingerprints but I cannot see my notebook integrated bluetooth there. I've also tried to enroll the notebook, it gives me an irk but it doesn't appear in the fingerprints anyway afterward. 
I've also tried to make the device discoverable, same result.

It would be great to create new automation in home assistant.
Could somebody experience the same? Do you have any hint?

Thanks!

**Comment by Terastar-Paperclip** [comment, marked as answer]
Windows laptops show up as `msft:cdp:0902` — DT explains this in [#222](https://github.com/ESPresense/ESPresense/discussions/222): "We can't tell them apart, but we can count the number in a room." So you can detect "a Windows device is in this room" but not identify a specific notebook by Bluetooth. For per-device tracking, you'd need a paired BLE beacon on the laptop. Cross-linking for visibility.
